recipe/meta.yaml

```diff
@@ -11,15 +11,15 @@ source:
   patches:
 
 build:
-  number: 0
+  number: 1
   skip: true  # [win]
   script:
   # install buf
     - set -e
     - set -x
     - export SETUPTOOLS_SCM_PRETEND_VERSION={{ version }}
   # this needs to match the upstream release commit SHA for extension lookup to work
-    - export SETUPTOOLS_SCM_PRETEND_HASH="109f932c4"
+    - export SETUPTOOLS_SCM_PRETEND_HASH="7c111322d"
```

Comment:
Yeah -- I added a test for 0.5.0 to make sure we could _install_ extensions but that, of course, kept passing since the previous hash was valid.  Just added in a check to also _load_ the extension which I believe should fail if there's a mismatch.